CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ cmake_minimum_required(VERSION 2.8.9)
 
 foreach(p
   CMP0054 # CMake 3.1
+  CMP0020 # CMake 3.0.2
```

Comment:
I think this should be

CMP0020 # CMake 2.8.11

Otherwise it looks good and works without problem on Windows.